templates/bands-development.ini.erb

```diff
@@ -4,6 +4,12 @@
 
 
 [dates]
-<% if @stage_agreement_deadline %>
```

Comment:
We remove these checks (and set actual dates in plugin_bands.pp) because we want the ability to set deadlines to `""`. which removes steps from the band checklist.